Sites/PR4/registernow.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>GD2 PR4</title>
+    <link rel="stylesheet" href="Styles.css">
+</head>
+
+
+<body>
+    <div class="containerwhite clearfix">
+
+
+        <h1><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 146.19 78.17">
+
+                <title>Asset 1</title>
+                <g id="Layer_2" data-name="Layer 2">
+                    <g id="Layer_1-2" data-name="Layer 1"><text class="alternate" transform="translate(0.22 30.85)">
+                            <tspan class="transparentblack">AB</tspan>
+                            <tspan class="transparentblack" x="0" y="30">C</tspan>
+                            <tspan class="blue" x="26.68" y="30">Design</tspan>
+                        </text></g>
+                </g>
+            </svg></h1>
+        <div id="closer">
+
+            <a href="index.html">Home</a> &nbsp / &nbsp <a href="learnmore.html">Learn More</a> &nbsp / &nbsp <a
+                href="#" class="bold">Register
+                Now </a>
+            <br><br>
+            <div class=smallspace>
+                <h2>CONTACT US!</h2> <br><br>
+            </div>
+            <div class=smallspace>
+                <label for="name"> Name : <span type="text" id="name" name="name">
+                        <div class=smallspace>
+                            <input type="text" id="name" name="name">
+                        </div> <br><br>
+                        <div class=smallspace>
+                            <label for="email"> Email Address : <span type="text" id="email" name="email">
+                                    <div class=smallspace>
```

Comment:
Your classname needs to be wrapped in double quotes
```suggestion
                                    <div class="smallspace">
```